static/sy.css

```diff
@@ -0,0 +1,54 @@
+@import url('https://fonts.googleapis.com/css?family=Montserrat:400,700|Open Sans:400,800');
+body {
```

Comment:
styles for the file should be either left without changes or inherited from bootstrap.